slp2mp4/gui_test.py

```diff
@@ -0,0 +1,289 @@
+import tkinter as tk
+from tkinter import filedialog
```

Comment:
While `tkinter` is a part of python's standard library, it isn't necessarily installed by default. There needs to be some kind of check that tkinter is installed, and have the program quit out and recommend how to install if it isn't.